docs/reference/mapping/types/parent-join.asciidoc

```diff
@@ -286,21 +286,70 @@ GET my-index-000001/_search
       }
     }
   },
-  "script_fields": {
-    "parent": {
-      "script": {
-         "source": "doc['my_join_field#question']" <3>
-      }
+  "runtime_mappings": {
+    "question": {
```

Comment:
What do you think about keeping this field as `parent` to align with the other fields in this request?

Reply:
Yeah, I can do that. I think `question` might be a better name in real life because its the name of the relation, but `parent` is what the rest of the example uses and it does highlight the "parent join" field.